test/functional/ui/spell_spec.lua

```diff
@@ -15,7 +15,6 @@ describe('combining long lines', function()
     screen = Screen.new(80, 4)
     screen:attach()
     screen:set_default_attr_ids( {
-      [0] = {bold=true, foreground=Screen.colors.Blue},
```

Comment:
Typically we make the window have enough rows so at least one empty line is visible.